eventstream_rpc/include/aws/eventstreamrpc/EventStreamClient.h

```diff
@@ -386,7 +386,7 @@ namespace Aws
             virtual bool OnStreamError(Crt::ScopedResource<OperationError> operationError, RpcError rpcError);
         };
 
-        enum AWS_EVENTSTREAMRPC_API ResultType
+        enum ResultType
```

Comment:
What is going on in this file?  The import/export macro usage seems really inconsistent.

Reply:
I don't know what else is going on in this file, but doxygen was only complaining about this one, and no other enums has an import/export, which makes sense because it's not a symbol a loader would need to find at runtime, it's just a value hardcoded in a header